blocks/hero/hero.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Variant Sizes: xlarge, large (Default), medium, small
+ */
+export default function decorate() {
```

Comment:
I see a lot of changes to the markup here, some guidance on best practices:
1. Use the "block" that is passed to decorate ( passed here: https://github.com/aemsites/esri/blob/105c78c8f0965c16248dccc890172477bfa40554/scripts/aem.js#L632)
2. Do not change the main structure (.section > .<block_name>-wrapper > .<block name>)
3. Try to minimize changes to the markup: before making a change to the markup (adding a class/changing the default structure) make sure that it brings value.

Reply:
- Used "block" param.  
- Minimize markup changes. 
- Will need script to dynamically build a <video> element with <source src='video url'>. Similar to dynamically built <picture> element with queries.